src/cascadia/QueryExtension/ExtensionPalette.cpp

```diff
@@ -373,6 +373,23 @@ namespace winrt::Microsoft::Terminal::Query::Extension::implementation
         _queryBox().Focus(FocusState::Programmatic);
     }
 
+    void ExtensionPalette::_exportMessagesToClipboard(const Windows::Foundation::IInspectable& /*sender*/,
+                                                       const Windows::UI::Xaml::RoutedEventArgs& /*args*/)
+    {
+        std::wstring concatenatedMessages{};
+        for (const auto groupedMessage : _messages)
+        {
+            for (const auto chatMessage : groupedMessage)
+            {
+                concatenatedMessages += chatMessage.as<ChatMessage>()->MessageContent() + L"\n";
```

Comment:
```suggestion
                concatenatedMessages += chatMessage.as<ChatMessage>()->MessageContent() + L"\n\n";
```

Do we want to add an extra newline to better separate each message? Personal preference of mine

Reply:
Now that it is exporting to file, do you still have this preference?

Reply:
Personally yeah. I just prefer the format:
```
paragraph 1

paragraph 2
```

over...
```
paragraph 1
paragraph 2
```

Reply:
Given that the AI sometimes sends many messages at once (code with explanation, with more code and more explanation), and _each_ of those messages would be separated by 2 newlines if we do this, my current inclination is that one newline is enough. Happy to change it if there's a consensus though